### ISSUE TITLE
Activated abilities with cost reduction can't be properly copied

### ISSUE THREAD

**Issue by theelk801** [magefree/mage#5499]
Currently, if a copy is made of Loreseeker's Stone, Nemesis of Mortals or Pteramander, the cost reduction effect won't apply to the copy's activated ability. I'm not sure how to fix this if anyone has any ideas.

**Comment by JayDi85** [comment]
You are using originalId -- it's Id of original ability from first card. And it's will ba same for all copies. That's the reason.

If you want calc activations count then use abilityId and save/load to game state that's count.

**Comment by theelk801** [comment]
Right, I know why it doesn't work, I'm just not sure how to fix it.

**Comment by JayDi85** [comment]
That's works fine: remove all originalId and use the code:
```
    @Override
    public boolean applies(Ability abilityToModify, Ability source, Game game) {
        return abilityToModify.getSourceId().equals(source.getSourceId());
    }
```

![shot_190110_025239](https://user-images.githubusercontent.com/8344157/50934899-91bf1080-1483-11e9-9fa8-a6c8e63ebe98.png)

**Comment by theelk801** [comment]
Right, but then it could modify other activated abilities on the same permanent

**Comment by JayDi85** [comment]
Fixed `Loreseeker's Stone` via cae517e3105f9e3f2557c72dade71e6bedcc0f48 workaround. It's works for original, copies and gains abilities. You can use same approach for other cards.

![shot_190110_040801](https://user-images.githubusercontent.com/8344157/50937681-35adb980-148e-11e9-9827-7c7991d01dc4.png)

![shot_190110_040834](https://user-images.githubusercontent.com/8344157/50937686-39d9d700-148e-11e9-851d-d69e3a6ca325.png)
